src/App.js

```diff
@@ -1,10 +1,25 @@
-import react from 'react';
+import { Switch, Route } from 'react-router-dom';
 import Calculator from './components/calculator';
+import Header from './components/Header';
+import Quote from './components/QuotePage';
+/* eslint-disable-next-line */
```

Comment:
- [ ] Please remove this eslint disable rule. You can refer to [this link](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md) for an idea on how to fix it.